test_runner/regress/test_compaction.py

```diff
@@ -116,6 +116,51 @@ def test_pageserver_compaction_smoke(neon_env_builder: NeonEnvBuilder):
     assert vectored_average < 8
 
 
+def test_pageserver_gc_compaction_smoke(neon_env_builder: NeonEnvBuilder):
+    # Effectively disable the page cache to rely only on image layers
```

Comment:
I suppose you copy pasted it from the other test in the file. I think it's not needed any more as we have removed the materialized page cache already (which this is about I suppose).

Reply:
https://github.com/neondatabase/neon/pull/9723/commits/681fb1bbf78f967878ab8653d89cfb6b88e797c3